lib/BoundsUtilities.js

```diff
@@ -0,0 +1,46 @@
+// Returns a Float32Array representing the bounds data for the sphere
+// and box. Assumes that both bounds are centered about the same spot
+// and that the box min and max are equal distance in every dimension
+function boundsToArray( bx, sp ) {
+
+	const arr = new Float64Array( 4 );
```

Comment:
Is this right? I thought it was going to be a `Float32Array` of length 7.

Reply:
It should be. I was originally going to build the box and sphere arrays separately but realized they could be merged. It should also be a Float32Array (oops). Thanks for the catch!